conf/application.conf

```diff
@@ -218,7 +218,6 @@ csv {
   uploadSizeLimit = 104857600
 }
 
-sessionTimeout {
-  timeoutSeconds = 900
-  time-out-countdown-seconds = 120
-}
+session {
+  timeout = 15.minutes
```

Comment:
out of curiosity, how is that different to the previous version (bootstrap-play seems to allow both versions)? does this implicitly handle the time when the countdown appears?

Reply:
Not gonna lie I don't think there's a difference, I saw the new one in their [guide](https://github.com/hmrc/play-frontend-hmrc#customisation) and figured we may as well use the one they mention. And yeah, the dialog appears at 2 minutes left by default (so if you set the timeout below that the dialog will always be there)